.gitleaks.toml

```
@@ -0,0 +1,10 @@
+title = "MBI Scan"
+# Only using custom rules
+
```

Unsure if this will be enduring, but _something_ general seems more appropriate here.
```suggestion
title = "DASG Standard"

```

---

Given that this is a proof of concept and the naming of the config file is an easy lift, I think I'm going to prefer reserving a choice like this until we're creating a central config to deploy across multiple repositories

---

I think it is worth setting a more general title here. Let's keep it very boring and call it "gitleaks config".